src/App.js

```diff
@@ -1,35 +1,88 @@
-import React from 'react';
-
-
-let todoList =[
-  {
-   id: 1,
-    title: "Call your Mother Again",
-   },
-  {
-   id: 2,
-    title: "Water the plants",
-   },
-  {
-     id: 3,
-     title: "Find a Loft",
-   },
-   ];
-
-   function App() {
-    return (
-      <div style={{ textAlign: 'center' }}>
-        <h1>Todo List</h1>
-        <ul>
-          {todoList.map(item =>
-            <li key={item.id}>
-              <li>{item.title}</li>
-            </li>
-            )}
+import React, { useEffect } from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+// The directions speak about a Nav Eelement 
+import Nav from './Nav'
+import './TodoListItem.module.css';
+
+// #1 Here we added the new Nav component - but there's no nav element, and hey no worries here
+// Also go back and retracefor sake of airtable away just to make this work for now
+
+
+function App() {
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
   
-        </ul>
-      </div>
-    );
+// According to the directions 
+  const baseId = process.env.REACT_APP_AIRTABLE_BASE_ID;
+const url = `https://api.airtable.com/v0/${baseId}/Default`;
+
+fetch(url, {
+  method: 'GET',
+  headers: {
+    Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+    'Content-Type': 'application/json'
   }
-  
-  export default App;
\ No newline at end of file
+})
+.then(response => response.json())
+.then(data => {
+  // do something with the response data
+})
+.catch(error => {
+  // handle any errors
+});
+
+  useEffect(() => {
+		fetch(url)
+			.then((response) => response.json())
+			.then((data) => {
+				setTodoList(data.records);
+				setIsLoading(false);
+			});
+	});
+
+  useEffect(() => {
+    if (!isLoading) {
+      localStorage.setItem("savedTodoList", JSON.stringify(todoList));
+    }
+  }, [todoList, isLoading]);
+
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
+
+  const removeTodo = (id) => {
+    const removeNewToDo = todoList.filter((list) => list.id !== id);
+    setTodoList(removeNewToDo);
+  };
+
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <div>
+<Nav />
+<h4 style={{ backgroundColor: 'grey', height: '.09in'}}></h4>
+
+<h1 style={{ backgroundColor: 'black', height: '1in', color: 'white', padding: '1.5rem' }}>Todo List</h1>
+<h5 style={{ backgroundColor: 'red', height: '.02in', color: 'white', padding: '.5rem' }}></h5>
```

Comment:
h elements are reserved for headings. To create elements like borders, such as you've done here, please use a <div></div> element. HTML Semantics (the elements you use) are very important for people who use your website who may have disabilities. Screen readers navigate the web using html elements, and if a screen reader thinks that a red border is a heading it will create confusion/frustration for your user.